lib/domain/annoyingDomains.ts

```diff
@@ -2,6 +2,25 @@ import { combineIngredientNamesAndAmounts, getItemListFromSelector, getTitleFrom
 import cheerio from 'cheerio';
 import { annoyingToParseDomains } from "./selectors";
 
+interface JoyFoodSunshineIngredientArray {
+    ingredients: [{
+        uid: number,
+        amount: string,
+        unit: string,
+        name: string,
+        notes: string,
+        converted: {
+            [key: string]: {
+                amount: string,
+                unit: string,
+                unit_id: number
+            }
+        },
+        unit_id: number,
+        id: number
+    }]
+}
```

Comment:
Is this needed, or is it fine to just use `any` in this case?

Reply:
Yeah I like the interface! Makes it safer/easier with the downstream inferencing